src/esw/odrive/odrive.py

```diff
@@ -178,15 +161,10 @@ def get_vel_estimate_m_s(self, axis: str) -> float:
         return vel_est_m_s
 
     def has_errors(self) -> bool:
-        """Returns a boolean to show if there are ODrive errors.
-        Args:
-            None.
-
-        Returns:
-            A boolean that is True if there are errors.
-
-        Raises:
-            DisconnectedError: If Jetson is unable to communicate with ODrive.
+        """
+        Returns a boolean to show if there are ODrive errors.
+        :returns: a boolean that is True if there are errors.

```

Comment:
keep it consistent with how I have it with other ESW PRs where sentences are capitalized (aka change to ":returns: A boolean". they do something else in main but might as well keep it consistent in ESW code and especially in the same file